developer-docs-site/docs/standards/aptos-token-v2.md

```diff
@@ -175,6 +205,10 @@ public entry fun mint_token(creator: &signer) {
 }
 ```
 
+Creators should cautiously consider whether they should use `create_named_token` or `create_from_account` when building
+their custom collection/token. In general `create_from_account` is recommended as it allows for clean deletion if the
+tokens are burnt and generally, deterministic addresses for tokens are not always necessary thanks to indexing services.
```

Comment:
I don't completely understand what "clean deletion" means. And why `create_from_account` supports clean deletion but not `create_named_token`?

Reply:
If `create_from_account` is strictly better, then what's the advantage of using `create_named_token`?